libsupport/include/katana/URI.h

```diff
@@ -24,10 +24,12 @@ class KATANA_EXPORT Uri {
   static Result<Uri> MakeFromFile(const std::string& str);
   /// Append a '-' and then a random string to input
   static Result<Uri> MakeRand(const std::string& str);
+
   static std::string JoinPath(const std::string& dir, const std::string& file);
 
-  /// Return the base64 (url variant) encoded version of this uri
-  std::string Encode() const;
+  // Decode returns the raw bytes represented by a URI encoded string.
+  static std::string Decode(const std::string& uri);
```

Comment:
While I understand this comment, when I read "raw bytes" I'm thining `std::vector<uint8_t>`.

Reply:
I mean what is a string but a buffer of bytes? :p